change/@azure-msal-browser-95141422-87f1-4a7b-a916-4a5ed71c293e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Add `hydrateCache()` API support",
+  "packageName": "@azure/msal-browser",
+  "email": "sameera.gajjarapu@microsoft.com",
```

Comment:
@tnorling if you're driving the commit do you want to add your name here?  Up to you, I'm OK to leave Sameera on this one if you'd prefer.

Reply:
She did most of the work so I'll leave her here. I'm just bringing it across the finish line :)